src/qml/editorwidgets/DateTime.qml

```diff
@@ -52,8 +55,9 @@ Item {
 
         Image {
           source: Style.getThemeIcon("ic_clear_black_18dp")
-          anchors.left: parent.right
-          visible: main.currentValue !== undefined && config['allow_null']
+          anchors.right: parent.right
+          anchors.verticalCenter: parent.verticalCenter
+          visible: ( value !== undefined || value === '' ) && config['allow_null']
```

Comment:
`( value !== undefined || value === '' )` isn't that equal to `value !== undefined`?

Reply:
Seemed like not. But I will recheck.